packages/java/endpoint/src/main/java/dev/hilla/crud/CrudRepositoryService.java

```diff
@@ -36,9 +37,30 @@ public CrudRepositoryService(R repository) {
         return getRepository().save(value);
     }
 
+    @Override
+    public T get(ID id) {
+        return getRepository().findById(id).orElse(null);
+    }
+
+    @Override
+    public boolean exists(ID id) {
+        return getRepository().existsById(id);
+    }
+
     @Override
     public void delete(ID id) {
         getRepository().deleteById(id);
     }
 
+    /**
+     * Counts the number of entities that match the given filter.
+     *
+     * @param filter
+     *            the filter, or {@code null} to use no filter
+     * @return
+     */
+    public long count(@Nullable Filter filter) {
+        return getRepository().count(toSpec(filter));
+    }
+
```

Comment:
I wonder if these three methods should be defined in the `ListRepositoryService` instead.

Reply:
I can move them

Reply:
ah but it becomes a bit odd then, because we don't want to make `ListService` any more complex, as you don't need the methods for listing items in a grid

Reply:
So the methods do not implement any interface, they just exist there, even though `get` and `exists` are in `CrudService` and need to match

Reply:
What does @sissbruecker think?

Reply:
I would just leave them where they are

Reply:
> ah but it becomes a bit odd then, because we don't want to make ListService any more complex, as you don't need the methods for listing items in a grid

What about creating another interface or super class (not sure what it would be called) with count() and exists() and get() that both `ListRepositoryService` and `CrudRepositoryService` would implement?

Reply:
Well, it is a little bit vague what are we trying to solve here. Just saving developers from writing some extra lines of code? In that case, I'd put whatever "reads" something from the data storage in the `ListRepositoryService`, and whatever manipulates data in the `CrudREpositoryService`. The interfaces `ListService` and `CrudService` still can remain as simple as possible, just having whatever necessary for their job. 

Another approach is to remain minimal, and ask who needs what? In that case who needs `count`? We can remove it. 

Are there other considerations?

Reply:
Ok, does the new version make sense?